docs/ContributingWizard.md

```diff
@@ -0,0 +1,147 @@
+# Wizard
+
+This guide shows how Wizard Web UI is structured, and walks through the approach for making changes.
+
+> Not every addition to the bicep will need representation in the Wizard Web UI - but we find that most will.
+
+## Overview
+
+### Presets
+
+The web app is preset driven, different presets will change the values of the controls in the tabs. Most controls map directly to a parameter for the main.bicep code file.
+
+The UI is initially populated with values from the config.json file. Selecting a preset then overlays specific values in the UI.
+
+![overlay preset json](images/wiz-overlay-config.png)
+
+### Guidance in the UI
+
+Whilst not being a teaching resource, adding UI hints, validation and links to the Azure documentation are all  important to the configuration experience.
```

Comment:
- [ ] Double space
- [ ] Is it worth mentioning that we want hints to be brief, with a preference for links to documentation?